src/core/function.js

```diff
@@ -1131,3 +1151,8 @@ var PostScriptCompiler = (function PostScriptCompilerClosure() {
 
   return PostScriptCompiler;
 })();
+
+exports.PDFFunction = PDFFunction;
+exports.PostScriptEvaluator = PostScriptEvaluator;
+exports.PostScriptCompiler = PostScriptCompiler;
+}));
```

Comment:
Yes, `node make lint` was throwing "ERROR:The non-exported symbol is referred: isPDFFunction from pdfjs/core/function used in pdfjs/core/evaluator". I think we need to change travis to execute that instead.